### DISCUSSION TITLE
Is Statiq.Markdown included in Statiq.Framework?

### DISCUSSION THREAD

**Discussion by RapidOwl** [statiqdev/Discussions#12] [Q&A] [answered]
I'm following the tutorial (https://statiq.dev/framework/) and the markdown part doesn't work. Do I need to install another package?

I'm using version 1.0.0-beta.19.

![image](https://user-images.githubusercontent.com/1280948/89918270-b9db9c00-dbf1-11ea-912a-fa9edd60f108.png)

**Comment by daveaglick** [comment, marked as answer]
Whops! I'll need to update the quick start. While [Statiq.Markdown](https://www.nuget.org/packages/Statiq.Markdown) is part of the overall Statiq Framework project, it's not included as a direct reference since we don't want to make any assumptions about what kind of generator is being built.

You'll need to add the [Statiq.Markdown](https://www.nuget.org/packages/Statiq.Markdown) package and that'll make the `RenderMarkdown` module available (likewise, [Statiq.Razor](https://www.nuget.org/packages/Statiq.Razor) will provide the Razor rendering module, etc.).

**Reply by daveaglick** [reply]
FYI - I've now updated the documentation to clarify this

**Reply by RapidOwl** [reply]
Brilliant! Thank you! I'll give it another go when I get a moment.
